eoscostarica/guias.eoscostarica.io
# Add to developers guidelines investigation IPFS

docs(IPFS-investigation): add to developers guidelines investigation IPFS 

### What does this PR do?

- resolves #307 

### Steps to test
1. Go to [Developers Guidelines](http://localhost:3000/docs/pautas-para-codigo-abierto)
1. Then click on IPFS (Interplanetary File System)
1. See the new doc

#### CheckList
